src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -945,9 +936,6 @@ void Transformations::PostLpt() {
     }
 #endif  // OPENVINO_ARCH_X86_64
 
-    CPU_REGISTER_PASS_COMMON(postLPTPassManager, ov::pass::transpose_sinking::TSShapeOfForward);
-    CPU_REGISTER_PASS_COMMON(postLPTPassManager, StatefulSDPAFusion);
-    CPU_REGISTER_PASS_X64(postLPTPassManager, ov::intel_cpu::SDPAFuseTransposeReshape);
```

Comment:
As you explained off-line, `SDPAFuseTransposeReshape` has strange regression in some cases, since being removed here.
Why not leave this line as commented and add a comment about the reason or TODO?

Reply:
Done.